src/fem/dof_cell_accessor.hh

```diff
@@ -0,0 +1,96 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ *
+ * @file    dof_cell_accessor.hh
+ * @brief
+ * @author  David Flanderka
+ */
+
+#ifndef DOF_CELL_ACCESSOR_HH_
+#define DOF_CELL_ACCESSOR_HH_
+
+#include "mesh/accessors.hh"
+
+/**
+ * Cell accessor allow iterate over DOF handler cells.
+ *
+ * Iterating is possible over different ranges of local and ghost elements.
+ */
+class DofCellAccessor {
```

Comment:
Name "DofCellAccessor" is maybe not exactly reflecting the purpose of this class ("Dof" here stands for DofHandler but usually it denotes "degree of freedom"). Suggestions: DHCellAccessor, DHElementAccessor.